### ISSUE TITLE
Bug NNBD: type 'Null' is not a subtype of type 'String' in type cast

### ISSUE THREAD

**Issue by hoc081098** [lejard-h/google_maps_webservice#93]
- Version
```yaml
google_api_headers: ^0.2.0-nullsafety.0
```

- Log
```
[VERBOSE-2:ui_dart_state.cc(186)] Unhandled Exception: type 'Null' is not a subtype of type 'String' in type cast
#0      _$PredictionFromJson (package:google_maps_webservice/src/places.g.dart:358:20)
#1      new Prediction.fromJson (package:google_maps_webservice/src/places.dart:934:7)
#2      _$PlacesAutocompleteResponseFromJson.<anonymous closure> (package:google_maps_webservice/src/places.g.dart:341:37)
#3      MappedListIterable.elementAt (dart:_internal/iterable.dart:411:31)
#4      ListIterator.moveNext (dart:_internal/iterable.dart:340:26)
#5      new _GrowableList._ofEfficientLengthIterable (dart:core-patch/growable_array.dart:188:27)
#6      new _GrowableList.of (dart:core-patch/growable_array.dart:150:28)
#7      new List.of (dart:core-patch/array_patch.dart:50:28)
#8      ListIterable.toList (dart:_internal/iterable.dart:211:44)
#9      _$PlacesAutocompleteResponseFromJson (package:google_maps_webservice/src/places.g.dart:342:14)
#10     new PlacesAutocompleteResponse.fromJson (package:go<…>
```

**Comment by hoc081098** [comment]
Seem `id` property does not exists  https://github.com/lejard-h/google_maps_webservice/blob/4dc939e745ab7aca502cc680dae90526ed2ba006/lib/src/places.dart#L901

**Comment by bhanuka96** [comment]
I am getting this error

```
../../../../dev/flutter/.pub-cache/hosted/pub.dartlang.org/google_maps_webservice-0.0.19/lib/src/utils.dart:61:27: Error: The argument type 'String' can't be assigned to the parameter type 'Uri'.
 - 'Uri' is from 'dart:core'.                                           
    return httpClient.get(url, headers: headers);                       
                          ^                                             
../../../../dev/flutter/.pub-cache/hosted/pub.dartlang.org/google_maps_webservice-0.0.19/lib/src/utils.dart:74:28: Error: The argument type 'String' can't be assigned to the parameter type 'Uri'.
 - 'Uri' is from 'dart:core'.                                           
    return httpClient.post(url, body: body, headers: postHeaders); 
```

**Comment by hoc081098** [comment]
> I am getting this error
> 
> ```
> ../../../../dev/flutter/.pub-cache/hosted/pub.dartlang.org/google_maps_webservice-0.0.19/lib/src/utils.dart:61:27: Error: The argument type 'String' can't be assigned to the parameter type 'Uri'.
>  - 'Uri' is from 'dart:core'.                                           
>     return httpClient.get(url, headers: headers);                       
>                           ^                                             
> ../../../../dev/flutter/.pub-cache/hosted/pub.dartlang.org/google_maps_webservice-0.0.19/lib/src/utils.dart:74:28: Error: The argument type 'String' can't be assigned to the parameter type 'Uri'.
>  - 'Uri' is from 'dart:core'.                                           
>     return httpClient.post(url, body: body, headers: postHeaders); 
> ```

Trying this workaround:
```yaml
dependency_overrides:
  google_maps_webservice:
    git:
      url: https://github.com/Flutter-Dart-Open-Source/google_maps_webservice.git
      ref: 98dc0bd451abb041a17f3efd254d5b983cbdb923

**Comment by lejard-h** [comment]
@hoc081098 I just published `0.0.20-nullsafety.1` version with a fix, ping me here if you still have the issue or not.

@bhanuka96 if you use the null safety version of the package, you should not have the issue. If you still have it, please open an other issue since it's unrelated to the one originally reported
